src/adapters/inbound/telegram_bot/bot.py

```diff
@@ -0,0 +1,273 @@
+from enum import Enum
+
+from telegram import (
+    ForceReply,
+    InlineKeyboardButton,
+    InlineKeyboardMarkup,
+    Update,
+)
+from telegram.ext import (
+    CallbackQueryHandler,
+    CommandHandler,
+    ContextTypes,
+    ConversationHandler,
+    MessageHandler,
+    filters,
+)
+
+from src.adapters.inbound.telegram_bot.controller import TelegramBotController
+from src.domain.ports.outbound.repositories.client import (
+    ClientRepositoryInterface,
+)
+from src.domain.ports.outbound.repositories.item import ItemRepositoryInterface
+from src.domain.ports.outbound.repositories.order import (
+    OrderRepositoryInterface,
+)
+from src.domain.use_cases.add_item import AddItemUseCase
+from src.domain.use_cases.cancel_order import CancelOrderUseCase
+from src.domain.use_cases.create_order import CreateOrderUseCase
+from src.domain.use_cases.remove_item import RemoveItemUseCase
+from src.domain.use_cases.set_inventory_quantity import (
+    SetInventoryQuantityUseCase,
+)
+
+
+class ConversationState(int, Enum):
+    WAITING_OPTION = 0
+    WAITING_ADD_ITEM = 1
+    WAITING_REMOVE_ITEM = 2
+    WAITING_SET_INVENTORY_QUANTITY = 3
+    WAITING_CREATE_ORDER = 4
+    WAITING_CANCEL_ORDER = 5
+
+
+class TelegramBotCommandHandler:
+    def __init__(
+        self,
+        item_repository=ItemRepositoryInterface,
+        order_repository=OrderRepositoryInterface,
+        client_repository=ClientRepositoryInterface,
+    ):
+        self.item_repository = item_repository
+        self.order_repository = order_repository
+        self.client_repository = client_repository
+
+    async def start_command(
+        self, update: Update, context: ContextTypes.DEFAULT_TYPE
```

Comment:
Aqui também não seria algo do tipo
```suggestion
        self, update: Update, context: ContextTypes = ContextTypes.DEFAULT_TYPE
```
?